extensions/wikia/UserProfilePageV3/UserIdentityBoxDiscussionInfo.class.php

```diff
@@ -36,37 +35,14 @@ public function getDiscussionAllPostsByUserLink() {
 	}
 
 	private function fetchDiscussionPostsNumber() {
-		$siteId = F::app()->wg->CityId;
 
-		$this->discussionActive = $this->checkDiscussionActive( $siteId );
+		$this->discussionActive = F::app()->wg->EnableDiscussions;
 		if ( $this->discussionActive ) {
-			$this->discussionPostsCount = $this->fetchDiscussionPostsCount( $siteId );
+			$this->discussionPostsCount = $this->fetchDiscussionPostsCount();
 			$this->discussionAllPostsByUserLink = "/d/u/{$this->userId}";
 		}
 	}
 
-	private function checkDiscussionActive( $siteId ) {
-		try {
-			$this->getDiscussionSitesApi()->getSite( $siteId );
-			return true;
-		} catch ( ApiException $e ) {
-			$this->logger->debug( 'Getting site caused an error',
-				[
-					'siteId' => $siteId,
-					'error' => $e->getMessage(),
-				] );
-		}
-
-		return false;
-	}
-
-	/**
-	 * @return SiteApi
-	 */
-	private function getDiscussionSitesApi() {
-		return $this->getDiscussionApi( SitesApi::class );
-	}
-
 	private function getDiscussionApi( $apiClass ) {
 		$apiProvider = Injector::getInjector()->get( ApiProvider::class );
 		$api = $apiProvider->getApi( self::DISCUSSION_SERVICE_NAME, $apiClass );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)